lib/ansible/modules/mount_facts.py

```diff
@@ -0,0 +1,637 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+
+DOCUMENTATION = """
+---
+module: mount_facts
+version_added: 2.18
+short_description: Retrieve mount information.
+description:
+  - Retrieve information about mounts from preferred sources and filter the results based on the filesystem type and device.
+options:
+  devices:
+    description: A list of fnmatch patterns to filter mounts by the special device or remote file system.
+    default: ~
+    type: list
+    elements: str
+  fstypes:
+    description: A list of fnmatch patterns to filter mounts by the type of the file system.
+    default: ~
+    type: list
+    elements: str
+  sources:
+    description:
+      - A list of sources used to determine the mounts. Missing file sources (or empty files) are skipped.
+      - The C(mount_points) return value contains the first definition found for a mount point.
+      - Additional mounts to the same mount point are available from  C(aggregate_mounts) (if enabled).
```

Comment:
```suggestion
      - Additional mounts to the same mount point are available from C(aggregate_mounts) (if enabled).
```
(remove extra space)

Reply:
Thanks, good catch.